pyproject.toml

```diff
@@ -1,6 +1,15 @@
 [project]
 name = "zeekscript"
-dynamic = ["version"]
+dynamic = [
+        "classifiers",
+        "dependencies",
+        "description",
+        "keywords",
+        "maintainers",
+        "readme",
+        "requires-python",
+        "version",
+]
```

Comment:
Hmm, this seems crossed to me. None of these are actually dynamic.

After looking a bit around: We're fully relying on the old setuptools approach. Seems more reasonable to drop the whole `[project]` section from `pyproject.toml` instead, or switching fully away from setup.py.